powdr-test/examples/serialized-inputs/src/main.rs

```diff
@@ -0,0 +1,33 @@
+use powdr::Session;
+
+#[derive(serde::Serialize, serde::Deserialize)]
+struct Data {
+    numbers: Vec<u32>,
+    sum: u32,
+}
+
+fn main() {
+    env_logger::init();
+
+    let some_data = Data {
+        numbers: vec![1, 2, 3, 4, 5],
+        sum: 15,
+    };
+
+    let s: String = "test".to_string();
+
+    let mut session = Session::builder()
+        .guest_path("./guest")
+        .out_path("powdr-target")
+        .chunk_size_log2(18)
+        .build();
+
+    session.write(&some_data);
+    session.write(&s);
+
+    // Fast dry run to test execution.
+    session.run();
+
+    // Uncomment to compute the proof.
+    //session.prove();
```

Comment:
I think it's fine to leave the proof out here to make this test faster, here we want to test inputs only anyway.

Reply:
but proof with initial memory inputs tested somewhere else?

Reply:
yea in the new riscv test added also here in this PR